pkg/healthcheck/sidecar.go

```diff
@@ -7,7 +7,7 @@ import (
 	corev1 "k8s.io/api/core/v1"
 )
 
-// HasExistingSidecars returns true if the pod spec already has the proxy init
+// HasExistingSidecars returns true if the name spec already has the proxy init
```

Comment:
The previous usage of _pod spec_ sounds better to me.

Reply:
That was an automatic refactor gone bad...